src/main/lib/logs.mjs

```diff
@@ -104,43 +104,15 @@ class Logs {
   // and returns the URL to access it. We need this to use electron-dl (which uses the electron native APIs)
   // to download the logs
   async getAllLogsURL (id) {
-    async function * concatStreams (readables) {
-      for (const readable of readables) {
-        for await (const chunk of readable) {
-          yield chunk
-        }
-      }
-    }
-
     if (!this.#logServer) {
       this.#logServer = Fastify()
```

Comment:
Why do you need a fasify server here?

Reply:
It's written in the comment above: https://github.com/platformatic/meraki/pull/421/files#diff-503458419384c686fc2914fedb2f76fca1aba0b3c2fce5c55707110ac43a7d59L103